### DISCUSSION TITLE
A32NX Latest Build (Github) Developer Version

### DISCUSSION THREAD

**Discussion by ghino25** [flybywiresim/aircraft#2374] [Help]
Hi Folks,

I have a question, If I want to get the latest A32NX Dev version from Github (https://github.com/flybywiresim/a32nx) and downloaded the master .zip file. Do I only need to copy the A32NX folder after it is extracted?

Regards,

**Comment by hotshotp** [comment]
please use the installer https://github.com/Externoak/A32NX-installer/releases/latest/download/A32NX_Downloader.zip
